gnomad_mitochondria/utils/filtering.py

```diff
@@ -171,57 +170,3 @@ def filter_genotypes(input_mt: hl.MatrixTable) -> hl.MatrixTable:
     )
 
     return input_mt
-
-
-def get_indel_expr(input_mt: hl.MatrixTable) -> hl.expr.BooleanExpression:
```

Comment:
(basic question) why is this being moved to /utils/annotations.py ? Is it used to add an annotation to a table later?

Reply:
yeah it's just adding information on whether or not it's a PASS indel, but doesn't actual filter anything, the functions in utils/filtering.py are ones that actually remove samples/variants/genotypes